src/services/sdk.ts

```diff
@@ -99,14 +117,39 @@ const buildPublication = (batchData: BatchFileData): Publication => {
   return {
     announcementType: core.announcements.AnnouncementType.Broadcast,
     fileUrl: batchData.url.toString(),
-    fileHash: "0x" + batchData.hash,
+    fileHash: batchData.hash,
   };
 };
 
+// TODO: move this dispatch code into a callback for subscribe
```

Comment:
This should really be attached to a story unless you're planning to address it in a follow-up